modules/library/main/src/test/java/org/geotools/styling/SLDTransformerTest.java

```diff
@@ -1405,10 +1415,28 @@ public void testLabelCDataMid() throws Exception {
         StyledLayerDescriptor sld = buildSLDAroundSymbolizer(ts);
         
         String xml = transformer.transform(sld);
-        assertTrue(xml.contains("<sld:Label><![CDATA[a  bc]]>" + NEWLINE +
-                "                            <ogc:PropertyName>myProperty</ogc:PropertyName>" + NEWLINE
-                + "                        </sld:Label>"));
+        Document doc = buildTestDocument(xml);
+
+        //assertXpathEvaluatesTo("a  bc", "normalize-space(//sld:Label/text()[1])", doc);
```

Comment:
can we remove commented out test?

Reply:
So, that is xmlunit being wrong. If xmlunit gets fixed, we ought to be able to uncomment it; I'll add a comment to make this more clear.